data/virt_autotest/virt-tests.conf

```diff
@@ -0,0 +1,136 @@
+[vmi_lifecycle_test]
+GINKGO_FOCUS=vmi_lifecycle_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vm_test]
+GINKGO_FOCUS=vm_test.go
+GINKGO_SKIP=test_id:3312
+EXTRA_OPT=-apply-default-e2e-configuration
```

Comment:
It looks like current this conf content is fixed. I am not sure whether it is going to change at some point, for example, you are going to add more tests or adjust test behavior. Can this implementation be make more flexible ? For example, you can easily pick up the wanted tests to be run instead of all ?

Reply:
Move this support into future PR. Created a ticket to track - https://progress.opensuse.org/issues/109256